src/utils/include/com/amazonaws/kinesis/video/utils/Include.h

```diff
@@ -10,6 +10,8 @@ extern "C" {
 
 #pragma once
 
+#include <syslog.h>
```

Comment:
You can move this to https://github.com/awslabs/amazon-kinesis-video-streams-pic/blob/master/src/common/include/com/amazonaws/kinesis/video/common/CommonDefs.h